ChatSecure/Classes/View Controllers/OTRMessagesViewController.m

```diff
@@ -1945,4 +1929,37 @@ - (IBAction)didPressMigratedSwitch {
     }
 }
 
+// If we find any incoming migration link messages, show the "your friend has migrated" header
+// to allow the user to start chatting with the new account instead.
+- (void) checkRangeForMigrationMessage:(NSRange)range {
```

Comment:
Well, then we might have another problem, because messageAtIndexPath is called ALL THE TIME by the other code, right?

Anyway, I moved the coded downstream instead. I still think the delegate is needed though (the last change still present here) for in-app link handling. But I guess I can move that into downstream as well.